lern-data-products/src/main/scala/org/sunbird/lms/job/report/BaseReportsJob.scala

```diff
@@ -78,6 +78,9 @@ trait BaseReportsJob {
         case "s3" =>
           sc.hadoopConfiguration.set("fs.s3n.awsAccessKeyId", AppConf.getConfig(reportsStorageAccountKey));
           sc.hadoopConfiguration.set("fs.s3n.awsSecretAccessKey", AppConf.getConfig(reportsStorageAccountSecret));
+        case "oci" =>
+          sc.hadoopConfiguration.set("fs.s3n.awsAccessKeyId", AppConf.getConfig(reportsStorageAccountKey));
+          sc.hadoopConfiguration.set("fs.s3n.awsSecretAccessKey", AppConf.getConfig(reportsStorageAccountSecret));
```

Comment:
Code is same for s3 and oci. Please use the pipe in case to match multiple values. 
Sample code:
`code match {
  case "1" | "2" => // whatever
  case "3" =>
}`

Reply:
made changes as suggested